### ISSUE TITLE
refreshenv.cmd doesn't work with nushell

### ISSUE THREAD

**Issue by aminya** [nushell/nushell#4624]
### Describe the bug

There is a script called `refreshenv.cmd` that is automatically installed if chocolatey is installed on Windows. What it does is refresh the environment variables of the currently running shell. It works with cmd and PowerShell, but not with nutshell.

https://github.com/chocolatey/choco/blob/master/src/chocolatey.resources/redirects/RefreshEnv.cmd
  

### How to reproduce

Run `refreshenv.cmd`

### Expected behavior

There should be a way to refresh the environment variables of the current running shell without restarting it.

### Screenshots

N/A

### Configuration

N/A

**Comment by fdncred** [comment]
Someone may be able to port this batch file since nushell has the `let-env` command. Of course, it would only work on windows because it's grabbing the env vars from the Windows registry in order to reset them.

**Comment by cdecompilador** [comment]
Yeah, but a better solution would be to just have an alternative command to really refresh all the environment variables, because at the moment event instanciating a new `nushell` doesn't reload the env, you have to restart that terminal session.

**Comment by fdncred** [comment]
this is what modules and overlays are for. we would just need to show the root overlay which would hide any other env vars, theoretically.
